src/output-json.c

```diff
@@ -1183,27 +1182,47 @@ OutputInitResult OutputJsonInitCtx(ConfNode *conf)
             }
 
             if (SCConfLogOpenRedis(redis_node, json_ctx->file_ctx) < 0) {
+                SCFree(json_ctx->file_ctx->sensor_name);
                 LogFileFreeCtx(json_ctx->file_ctx);
```

Comment:
`LogFileFreeCtx` will free this as well. Potentially leading to a double-free.  In practice in can never happen as all errors inside `SCConfLogOpenRedis` are fatal, but this addition is not needed.

Reply:
Will correct.